{
    "type": "bundle",
    "id": "bundle--52a956ac-7ed5-435b-96f3-db2b5ea57240",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--c52d7cd7-fffd-463b-803f-3c851ab1368e",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-05-21T00:43:09.15653Z",
            "modified": "2025-05-21T00:43:09.15653Z",
            "name": "CVE-2025-37906",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nublk: fix race between io_uring_cmd_complete_in_task and ublk_cancel_cmd\n\nublk_cancel_cmd() calls io_uring_cmd_done() to complete uring_cmd, but\nwe may have scheduled task work via io_uring_cmd_complete_in_task() for\ndispatching request, then kernel crash can be triggered.\n\nFix it by not trying to canceling the command if ublk block request is\nstarted.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-37906"
                }
            ]
        }
    ]
}